package net.particify.arsnova.core.service.exceptions;

public class UserAlreadyExistsException extends RuntimeException {
}
